src/doc-store/Store/Document.cs

```diff
@@ -68,7 +68,8 @@ public StoreDocument()
         }
         public StoreDocument(Document document)
         {
-            this.Client = document.Client;
+            //change client later to come from auth token instead of request body.
+            this.Client =  document.Client ?? "1337";
```

Comment:
maybe use a constant for the default since u use it multiple times - easier to get rid of it later